flink-runtime-web/new-web-dashboard/README.md

```diff
@@ -0,0 +1,60 @@
+Flink Runtime Web
+=================
+[![Travis branch](https://travis-ci.com/vthinkxie/flink-runtime-web.svg?branch=master)](https://travis-ci.com/vthinkxie/flink-runtime-web)
```

Comment:
Can you remove this line from the file? once merged to Flink, this build status will be invalid.